website/example/Select.demo.js

```diff
@@ -0,0 +1,35 @@
+// import React from 'react';
```

Comment:
Why is the code commented?

Reply:
Beacause I wasn't able to make it works.
I will need help from you for this.